### PR TITLE
BDD100K : Test Files

### PR DESCRIPTION
**Pull request by ananno**
## Motivation and Context

Adding unit testing file for bdd100k.

## Description

To automatic unit testing of both object detection and segmentation dataset classes this test files is implemented following blueoil test framework.

## How has this been tested?

`tox -e pytest`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ananno** [comment]
@iizukak San:
I began to separate #399 into multiple smaller PRs. This is the 5th one. Please review.

**Comment by ananno** [comment]
@tkng, @iizukak San:
I've applied isort as requested. Please review one more time.

**Comment by iizukak** [comment]
@ananno Please solve the conflicts.

**Comment by ananno** [comment]
@iizukak San: Can you review this one more time.おねがいします

**Comment by iizukak** [comment]
@ananno 
Please discuss about fixture image size in this issue.
https://github.com/blue-oil/blueoil/issues/700

**Comment by ananno** [comment]
Commit #796f705 fixes Issue #700

**Comment by ananno** [comment]
> @ananno
> Why both `100.jpg` and `1000.png` are exist?

@iizukak San:
You were right. We actually don't need 2 image files. I've removed one and use links to one image file `dummy_image` where necessary.

Now the fixture directory `BDD100K` becomes `72KB`. 😄 

> NOTE : I've tested with a 0 byte file but it didn't work as `PIL` package actually try to open the image.

**Comment by iizukak** [comment]
@ananno 
I think `BDD100K/dummy_image` need filename extension...

**Comment by iizukak** [comment]
@ananno ping

**Comment by ananno** [comment]
@iizukak San:
I've updated the filename for `dummy_image` and also updated the corresponding linked files paths. Please review one more time.
N.B. : The automatic test failed due to no space left in the device.

File: lmnet/tests/lmnet_tests/datasets_tests/test_bdd100k.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    train_image_files, train_label_files = train_iterator.feed()
+    assert train_image_files.shape[0] == batch_size
+    assert train_label_files.shape[0] == batch_size
+
+    test_image_files, test_label_files = test_iterator.feed()
+    assert test_image_files.shape[0] == batch_size
+    assert test_label_files.shape[0] == batch_size
+
+    print("file annotations are ready")
```
Comment (tsawada) [inline lmnet/tests/lmnet_tests/datasets_tests/test_bdd100k.py]: Don't print anything within tests

File: lmnet/tests/lmnet_tests/datasets_tests/test_bdd100k.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
```
Comment (tsawada) [inline lmnet/tests/lmnet_tests/datasets_tests/test_bdd100k.py]: same

File: lmnet/tests/lmnet_tests/datasets_tests/test_bdd100k.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+if __name__ == '__main__':
```
Comment (tsawada) [inline lmnet/tests/lmnet_tests/datasets_tests/test_bdd100k.py]: I think we shouldn't prepare any main function within `pytest` test scripts. might be wrong.

**Pull request merged**